rmgpy/rmg/main.py

```diff
@@ -1415,20 +1427,26 @@ def generateCanteraFiles(self, chemkinFile, **kwargs):
 
     def initializeReactionThresholdAndReactFlags(self):
         numCoreSpecies = len(self.reactionModel.core.species)
+
+        # Number of families in RMG
+        len_families = len(getDB('kinetics').families)
+
         if self.filterReactions:
-            self.unimolecularReact = np.zeros((numCoreSpecies),bool)
-            self.bimolecularReact = np.zeros((numCoreSpecies, numCoreSpecies),bool)
-            self.unimolecularThreshold = np.zeros((numCoreSpecies),bool)
-            self.bimolecularThreshold = np.zeros((numCoreSpecies, numCoreSpecies),bool)
+            self.unimolecularReact = np.zeros((numCoreSpecies, len_families) ,bool)
+            self.bimolecularReact = np.zeros((numCoreSpecies, numCoreSpecies, len_families),bool)
+            self.unimolecularThreshold = np.zeros((numCoreSpecies, len_families),bool)
+            self.bimolecularThreshold = np.zeros((numCoreSpecies, numCoreSpecies, len_families),bool)
             if self.trimolecular:
-                self.trimolecularReact = np.zeros((numCoreSpecies, numCoreSpecies, numCoreSpecies), bool)
-                self.trimolecularThreshold = np.zeros((numCoreSpecies, numCoreSpecies, numCoreSpecies), bool)
+                self.trimolecularReact = np.zeros((numCoreSpecies, numCoreSpecies, numCoreSpecies, 
+                    len_families), bool)
+                self.trimolecularThreshold = np.zeros((numCoreSpecies, numCoreSpecies, numCoreSpecies, 
+                    len_families), bool)
```

Comment:
It might be better to line this up with the first numCoreSpecies. Not totally sure though.

Or just keep it on one line, like line 1449.